hols/todo/backend/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1,21 @@
+## Microprofile server properties
+server.port=8080
+server.host=0.0.0.0
+#
```

Comment:
You can use newlines in properties file, can you remove some of the commented empty lines.